### PR TITLE
radosgw-admin: supplement help documents with 'mdlog autotrim'

### PR DESCRIPTION
**Pull request by Huber-ming**
Signed-off-by: Huber-ming <zhangsm01@inspur.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Review by dang — APPROVED** [review]

**Comment by dang** [comment]
jenkins test api

**Comment by dang** [comment]
jenkins test make check

**Review by dang — CHANGES_REQUESTED** [review]
You'll need to update src/test/cli/radosgw-admin/help.t to match the new update.

File: src/rgw/rgw_admin.cc [hunk 1]
```diff
@@ -239,6 +239,8 @@ void usage()
   cout << "  metadata list              list metadata info\n";
   cout << "  mdlog list                 list metadata log\n";
   cout << "  mdlog trim                 trim metadata log (use marker)\n";
+  cout << "  mdlog autotrim             auto trim metadata log\n";
+  cout << "  mdlog fetch                fetch metadata log\n";
```
Comment (cbodley) [inline src/rgw/rgw_admin.cc:243]: i'm not sure where `mdlog fetch` came from. i see a command for it, but there's no actual implementation:

```
$ grep MDLOG_FETCH rgw_admin.cc
  MDLOG_FETCH,
  { "mdlog fetch", OPT::MDLOG_FETCH },
```

**Review by cbodley — APPROVED** [review]

**Comment by cbodley** [comment]
@dang can you please re-review? i think this one can merge if you're satisfied

**Review by dang — APPROVED** [review]

**Pull request merged**